SIT.Manager/App.axaml.cs

```diff
@@ -161,33 +167,55 @@ private static ServiceProvider ConfigureServices
                 MaxConnectionsPerServer = 10
             });
 
-            services.AddResiliencePipeline<string, HttpResponseMessage>("default-pipeline", builder =>
+            services.AddHttpClient<NetworkToolsViewModel>(client =>
             {
-                builder.AddRetry(new RetryStrategyOptions<HttpResponseMessage>()
+                client.BaseAddress = new Uri("http://backend.sitcoop.org/");
```

Comment:
Should this string be a const at the top so it's easier to find and change in the future, assuming that it is the address we are meant to be using?